tools/js-optimizer.js

```diff
@@ -5730,12 +5730,13 @@ function safeHeap(ast) {
   traverseGeneratedFunctions(ast, function(func) {
     if (func[1] in SAFE_HEAP_FUNCS) return null;
     traverseGenerated(func, function(node, type) {
+      var heap, prt;
```

Comment:
`prt => ptr`

does closure even run on this file? that would surprise me but maybe I forgot something.

Reply:
Sorry, fixed.
I don't think closure runs here, but variables were defined twice in the same scope, so I though it is an appropriate change as I was fixing similar things in other places.